packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1425,6 +1425,47 @@ class OffsetLayer extends ContainerLayer {
       scene.dispose();
     }
   }
+
+  /// Capture an image of the current state of this layer and its children.
+  ///
+  /// The returned [ui.Image] has uncompressed raw RGBA bytes, will be offset
```

Comment:
Some of this documentation si shared with `toImage` and we should probably have a template for it.

These docs should mention something about the difference between this method and `toImage`.

Reply:
Added some more details. I don't think it would be right to describe this as containing uncompressed raw RGBA bytes since this is a GPU resident texture, right?

Reply:
Yeah it may be BGRA depending on the platform. It's less important what the byte format is here.